### PR TITLE
fix all the typos

### PR DESCRIPTION
**Pull request by trontrytel**
This is a breaking change so we should bump the version. I can update the CloudMicrophysics and Kinematic1D models when it happens. I think they should be the only ones affected

### CODE REVIEW

**Comment by nefrathenrici** [comment]
This looks good. Do you think that the `Chen2022_table_B3` should be array parameters instead of separate single values? We could get that change in before the minor release as well.

**Review by nefrathenrici — APPROVED** [review]

**Comment by trontrytel** [comment]
> This looks good. Do you think that the `Chen2022_table_B3` set of parameters should be a couple of array parameters instead of separate single values? We could get that change in before the minor release as well.

Yes, that would be nicer. So we should replace for example this:

```
[Chen2022_table_B3_As_coeff_1]
value = 0.00174079
type = "float"
description = "part of AS coefficient for ice terminal velocity parameterization - See Table B3 in Chen et al 2022 DOI: 10.1016/j.atmosres.2022.106171"

[Chen2022_table_B3_As_coeff_2]
value = 0.0378769
type = "float"
description = "part of As coefficient for ice terminal velocity parameterization - See Table B3 in Chen et al 2022 DOI: 10.1016/j.atmosres.2022.106171"

[Chen2022_table_B3_As_coeff_3]
value = 0.263503
type = "float"
description = "part of As coefficient for ice terminal velocity parameterization - See Table B3 in Chen et al 2022 DOI: 10.1016/j.atmosres.2022.106171"
```

with this:

```
[Chen2022_table_B3_As]
value = [0.00174079,  0.0378769, 0.263503]
type = "float"
description = "part of AS coefficient for ice terminal velocity parameterization - See Table B3 in Chen et al 2022 DOI: 10.1016/j.atmosres.2022.106171"
```
?

**Comment by nefrathenrici** [comment]
@trontrytel Yes, that looks perfect!

**Comment by trontrytel** [comment]
ok - I'll change it and test if I can use the elegant way in CloudMicrophysics. When do you want to do the release @nefrathenrici ?

**Comment by nefrathenrici** [comment]
We can do another release whenever is convenient. I think these changes can all be in one PR.
I have also found another typo: `rain_cross_section_size_relation_coefficient_dela`

File: src/parameters.toml [hunk 1]
```diff
@@ -935,155 +1008,60 @@ value = 0.115231
...
-[Chen2022_table_B1_a2_coeff]
-value = -0.263166
-type = "float"
-description = "a2 coefficinet for raindrop terminal velocity parameterization [mm^-bi]. See Table B1 in Chen et al 2022 DOI: 10.1016/j.atmosres.2022.106171"
-
-[Chen2022_table_B1_a3_coeff]
-value = 4.7178
-type = "float"
-description = "a3 coefficinet for raindrop terminal velocity parameterization [mm^-bi]. See Table B1 in Chen et al 2022 DOI: 10.1016/j.atmosres.2022.106171"
+description = "ai coefficinets for raindrop terminal velocity parameterization [mm^-bi]. See Table B1 in Chen et al 2022 DOI: 10.1016/j.atmosres.2022.106171"
```
Comment (nefrathenrici) [inline src/parameters.toml:1014]: There is a typo `coefficinets` -> `coefficients` in many descriptions. It would be good to fix those quickly as well!

**Review by nefrathenrici — APPROVED** [review]
Thank you for adding the descriptions!

File: src/parameters.toml (hunk 1 above)
Comment (trontrytel) [reply src/parameters.toml]: As long as I add less typos than I remove, things should continue to improve ;) Thank you for caching that. Should be fixed now.

**Pull request merged**